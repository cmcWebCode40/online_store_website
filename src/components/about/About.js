import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

const About = () => {
  return (
    <div className="about" id="about">
      <div className="about-card  px-1 py-3 ">
        <h2 className="px-2  mb-1">What we are all about</h2>
        <p className="px-2 ">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Saepe omnis, rerum neque illo sed assumenda fugit maxime,onseneque illo sed assumenda fugit maxime,onseneque illo sed assumenda fugit maxime,onsectetur adipisicing elit. Saepe omnis, rerum neque illo sed assumenda  eaque ratione </p>
        <span>
          <FontAwesomeIcon
            icon="play-circle"
            size="3x"
            style={{ marginRight: "1rem" }}
            color="#ee6a65"
          />
        </span>
      </div>
    </div>
  )
}

export default About
