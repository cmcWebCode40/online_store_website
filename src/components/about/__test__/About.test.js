import React from 'react';
import ReactDOM from 'react-dom';
import About from '../About';

import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

afterEach(cleanup);

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<About />, div);
});

it('renders button correctly', () => {
	const { getByTestId } = render(<About />);
	expect(getByTestId('heading')).toHaveTextContent('What we are all about');
});

it('should take a snapshot', () => {
	const { asFragment } = render(<About />);
	expect(asFragment(<About />)).toMatchSnapshot();
});
