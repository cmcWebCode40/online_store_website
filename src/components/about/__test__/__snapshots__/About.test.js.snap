// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="about"
    id="about"
  >
    <div
      class="about-card  px-1 py-3 "
    >
      <h2
        class="px-2  mb-1"
        data-testid="heading"
      >
        What we are all about
      </h2>
      <p
        class="px-2 "
      >
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Saepe omnis, rerum neque illo sed assumenda fugit maxime,onseneque illo sed assumenda fugit maxime,onseneque illo sed assumenda fugit maxime,onsectetur adipisicing elit. Saepe omnis, rerum neque illo sed assumenda.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
